source/utilities.tex

```diff
@@ -16959,6 +17011,49 @@
 supported\iref{basic.align}.
 
 \indexlibrary{\idxcode{common_type}}%
+\pnum
+Let:
+\begin{itemize}
+\item \tcode{CREF(A)} be \tcode{add_lvalue_reference_t<const remove_reference_t<A>{}>},
+\item \tcode{XREF(A)} denote a unary class template \tcode{T} such that
+  \tcode{T<U>} denotes the same type as \tcode{U} with the addition
+  of \tcode{A}'s cv and reference qualifiers, for a type \tcode{U} such
```

Comment:
How is that going to work? Suppose `U=int`; a class template `T<int>` can never be an "int". Is there a "::type" missing?

Reply:
This is the problematic wording I noted in the issue: "unary class template" here should be "unary alias template" or "unary type template". Marshall requested that I fix this via a followup LWG issue.